ui/app/components/clients/line-chart.js

```diff
@@ -42,24 +44,38 @@ export default class LineChart extends Component {
     return this.args.xKey || 'month';
   }
 
+  get upgradeData() {
+    const upgradeData = this.args.upgradeData;
+    if (!upgradeData) return null;
+    if (!Array.isArray(upgradeData)) {
+      console.error('upgradeData must be an array of objects containing upgrade history');
```

Comment:
nit: might consider changing these to `console.debug` instead

Reply:
thank you!! i was trying to remember this one. 😮‍💨